### PR TITLE
ZEPPELIN-844: Navbar menu should not show up if user is not authenticated

### PR DESCRIPTION
**Pull request by prabhjyotsingh**
### What is this PR for?
If a user in not authenticated, s/he should not see Navbar menu, or search bar, or import/create note book links.

### What type of PR is it?
[Bug Fix]

### Todos
* [x] - Hide all option from frontend

### What is the Jira issue?
[ZEPPELIN-844](https://issues.apache.org/jira/browse/ZEPPELIN-844)

### How should this be tested?
Change shiro conf to enable authentication. Now when home page is loaded you should not see any of the Navbar menu or option to create or import notebook

### Screenshots (if appropriate)
Before:
screen shot 2016-05-10 at 3 08 10 pm

After:
screen shot 2016-05-10 at 2 49 05 pm

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no 
* Does this needs documentation? no

### CODE REVIEW

**Comment by prabhjyotsingh** [comment]
Fails for `Job #4117.2` with `npm install --color=false` which is not related.

File: zeppelin-web/src/components/navbar/navbar.html [hunk 1]
```diff
@@ -94,8 +94,8 @@
         <li ng-if="!ticket">
             <button class="btn btn-default" data-toggle="modal" data-target="#loginModal" ng-click="showLoginWindow()" style="margin-left: 10px">Login</button>
         </li>
-        <li ng-show="ticket.principal != 'anonymous'">
-        	<button type="button" class="btn btn-default btn-xs" ng-click="logout()" tooltip-placement="bottom" tooltip="logout">{{ticket.principal}}</button>
+        <li ng-show="ticket" style="left: 5px;">
```
Comment (felixcheung) [inline zeppelin-web/src/components/navbar/navbar.html]: shouldn't it check what's in the ticket (principal)?

Comment (prabhjyotsingh) [inline zeppelin-web/src/components/navbar/navbar.html]: Fair point, thank you for pointing it out.

File: zeppelin-web/src/components/navbar/navbar.html [hunk 2]
```diff
@@ -94,8 +94,8 @@
         <li ng-if="!ticket">
             <button class="btn btn-default" data-toggle="modal" data-target="#loginModal" ng-click="showLoginWindow()" style="margin-left: 10px">Login</button>
         </li>
-        <li ng-show="ticket.principal != 'anonymous'">
-        	<button type="button" class="btn btn-default btn-xs" ng-click="logout()" tooltip-placement="bottom" tooltip="logout">{{ticket.principal}}</button>
+        <li ng-show="ticket.principal && ticket.principal!='anonymous'" style="left: 5px;">
+        	<button class="btn btn-default" ng-click="logout()" tooltip-placement="bottom" tooltip="logout">Logout</button>
```
Comment (felixcheung) [inline zeppelin-web/src/components/navbar/navbar.html]: is it handy to say "logout foo" (foo == username)
I'm not sure.

Comment (prabhjyotsingh) [inline zeppelin-web/src/components/navbar/navbar.html]: Agreed if logout button was coming in a separate menu (or different dropdown menu). Right now the UI already says `<li>{{truncatedUsername}}</li>
<li><button>Logout</button></li>`

**Comment by prabhjyotsingh** [comment]
Merging this if no more discussion.

**Pull request closed without merging**